recipe/meta.yaml

```diff
@@ -11,9 +11,11 @@ source:
   sha256: b7bb621a47135e1d351becb82cdbe21237c39d4fcbbeaec677e31db02cc68900
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - vak=vak.__main__:main
```

Comment:
for giggles, you may wish to add the following in `test:commands` (after `pip check`):

```yaml
- vak --version
- vak --help
```

Reply:
Ah right, that's what I should have done. Thank you

Reply:
Added another commit that does that